Java/Control Statements/Switch Expressions/6.3 Switch Expressions.md

```diff
@@ -0,0 +1,174 @@
+# Switch Expressions in Java
+
+<img src="https://java.tutorials24x7.com/uploads/2020-06-20/banner/tutorials24x7-switch-statements-and-expressions-in-java-banner.jpg" />
+A switch expression is an expression that is used when we need to use multiple conditions in our program. Now we can use this because if we use multiple If, else if, and else statements in the program, it will be difficult for the programmer and program to understand and write the code.
+So the syntax of the Switch Expressions is-:
+
+### Syntax-:
+<pre><code>
+
+switch(expression){
+	case one:
+		//code block
+		break;          //This will terminate the current case
+	case two:
+		//code block
+		break;
+	default:
+		//code block  (This will get executed if nothing is executed)
+		break;
+}
+</code></pre>
+-   If the break statement is not used then it will continue with other cases.
+-   Duplicate cases are not allowed in Switch Case.
+
+### Example-:
+
+<pre><code>
+switch(expression){
+	case one:
+		//code block
+		break;          //This will terminate the current case
+	case one:
+		//code block
+		break;
+	default:
+		//code block  (This will get executed if nothing is executed)
+		break;
+}
+</code></pre>
+**The above code will give an error**
+
+Switch statement example-:
+--------------------------
+
+<pre><code>
+import java.util.Scanner;
+
+public class Switch {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        String fruit = in.next();
+        switch(fruit){
+            case "Mango":
+                System.out.println("King of fruits");
+                break;
+            case "Apple":
+                System.out.println("Bye bye doctor");
+                break;
+            case "Orange":
+                System.out.println("Hey Vitamin C");
+                break;
+            case "Grapes":
+                System.out.println("Balls of green");
+                break;
+            default:
+                System.out.println("New object!!!");
+                break;
+        }
+        in.close();
+    }
+}
+</code></pre>
+
+Now the above code is how most of the people used the switch case statements, but Some more methods by which you can use them in our code are given below-:
+
+-   Using switch expressions in our Programme
+    ------------------------
+
+## 1\. **Returning Value using Arrow Label**
+
+Now it must be difficult writing all those break statements again and again so with the new versions of Java they introduced a new method/syntax of writing switch-case statements. The arrow pointer variable is beneficial to use instead of if-else statements as we don't need to use any break statements because if we forget to use them then the code will keep running and all left out cases will get executed.
```

Comment:
`because if we forget to use them` to `because if we forget to use them,` 
`then the code will keep running`  to `then the code keeps on running`